pkg/multimodelconfig/configmap.go

```diff
@@ -0,0 +1,78 @@
+package multimodelconfig
+
+import (
+	"encoding/json"
+	"k8s.io/api/core/v1"
+	"k8s.io/klog"
```

Comment:
we are trying to use [controller runtime logger](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/log) more consistently across the repo